Assets/LeapMotionModules/Hands/Shaders/BlendTesting.shader

```diff
@@ -0,0 +1,121 @@
+﻿Shader "Character Indicator"
```

Comment:
What is this shader for?  It seems to be just a regular alpha-cutoff shader right now, but looks like it might have been used for something else at some point?